### PR TITLE
webgpu: add flipLeftRight and rotate operators

### PR DESCRIPTION
**Pull request by xhcao**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5649)

### CODE REVIEW

**Comment by xhcao** [comment]
@qjia7 @haoyunfeix @axinging @gyagp, please take a look, thank you.

File: tfjs-backend-webgpu/src/kernels/rotate_webgpu.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+export class RotateProgram implements WebGPUProgram {
+    outputShape: number[] = [];
+    shaderKey: string;
+    dispatchLayout: {x: number[]};
+    dispatch: [number, number, number];
+    variableNames = ['x'];
+    uniforms: string;
+    uniformsWgsl: string;
+    workGroupSize: [number, number, number] = [64, 1, 1];
+    xShape: number[];
```
Comment (qjia7) [inline tfjs-backend-webgpu/src/kernels/rotate_webgpu.ts:34]: Same here.

**Review by qjia7 — APPROVED** [review]
LGTM with nits.

**Review by qjia7 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**